test/test_dataloader.py

```diff
@@ -13,7 +13,8 @@
 from torch.utils.data.dataset import random_split
 from torch.utils.data.dataloader import default_collate, ExceptionWrapper
 from common import TestCase, run_tests, TEST_NUMPY, IS_WINDOWS
-from common_nn import TEST_CUDA
+if __name__ == '__main__':
```

Comment:
Should add a default value for `TEST_CUDA` because it's used below.

Reply:
Would it be okay to put `TEST_CUDA = False` as default value here?

Reply:
I think it's okay.